shared/deps/serial/VirtIoChannel_guest_send_receive.py

```diff
@@ -95,7 +95,9 @@ def shake_hand(self, size=0, action="receive"):
                 raise ShakeHandError("Host didn't ACK the file size message.")
             return size
         elif action == "receive":
-            txt = self.receive(hi_msg_len)
+            txt = ''
+            while len(txt) < hi_msg_len:
+                txt += self.receive(hi_msg_len)
```

Comment:
It should be either:
```python
txt = b''
while not txt:
    txt = self.receive(hi_msg_len)
```
or
```python
txt = b''
while len(txt) < hi_msg_len:
    txt += self.receive(hi_msg_len - len(txt))
```

Reply:
@luckyh Yes, thanks. Updated to second one, but no `b` there, since it might need further handling on `receive` and `send` functions, I will fix py3 issues together later.